### DISCUSSION TITLE
Disable adding new page if page is empty

### DISCUSSION THREAD

**Discussion by semiherdogan** [spatie/ray#139] [Q&A]
I can see previous pages by pressing top arrow button, and clear page with "+" button see below,
but even i'm on the empty page when pressing "+" button creates new page (or using cmd + shift + k shortcut)
My question is is there any way to disabling creating new page if page is already empty?

![image](https://user-images.githubusercontent.com/5646041/104445911-c736f100-55aa-11eb-87ae-5a0759c786ac.png)

**Comment by semiherdogan** [comment]
Sorry, 
"+" button disabling, but keyboard shortcut creates new page.

**Reply by AdrianMrn** [reply]
Good call. This will be fixed in 1.2.5 (will release later today)
